Library/Homebrew/cmd/autoremove.rb

```diff
@@ -0,0 +1,53 @@
+# typed: false
+# frozen_string_literal: true
+
+require "formula"
+require "cli/parser"
+require "uninstall"
+
+module Homebrew
+  module_function
+
+  def autoremove_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `autoremove` [<options>]
+
+        Remove packages that weren't installed on request and are no longer needed.
+      EOS
+      switch "-n", "--dry-run",
+             description: "Just print what would be removed."
```

Comment:
```suggestion
             description: "List what would be uninstalled, but do not actually uninstall anything."
```

* this matches our wording for other `--dry-run` options